_posts/2015-09-30-buffer.md

```diff
@@ -12,19 +12,19 @@ In this post, we will discuss one of the most simple logic gates, a *buffer*.
 
 ![Buffer schematic]({{ site.url }}{{ site.baseurl }}/images/buffer_schematic.svg)
 
-A buffer takes an input signal, and bputs a signal in the same state as the
-input signal, the bput simply follows the input. For example, if the input
-signal changes from 0 to 1, the bput will eventually change from 0 to 1. We
-say the bput changes ‘eventually’ because the bput changes after some
+A buffer takes an input signal, and outputs a signal in the same state as the
```

Comment:
I was just amused by the susbstitution artefact `bput` (clearly from replacing `out`s with `b`s).